src/hotspot/share/oops/metadata.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
No more change in this file?

Reply:
I removed the copyright.
I tried to add an assert to CompresssedKlassPointers::encode_not_null but CDS calls this with ill-formed Klass* pointers so the tests crash.